test/helpers.h

```diff
@@ -69,39 +65,59 @@ class SimpleTokenizer : public Tokenizer {
         }
      */
 
-    /**
-     * Tokenize code
-     * @param code The code
-     * @param cpp Indicates if the code is C++
-     * @param configuration E.g. "A" for code where "#ifdef A" is true
-     * @return false if source code contains syntax errors
-     */
+    template<size_t size>
+    bool tokenize(const char (&code)[size],
```

Comment:
why is this a template. I fear that for almost each test code example there will be a separate function instantiation. And it just seems redundant.

Reply:
See https://github.com/danmar/cppcheck/pull/7427#issuecomment-2775888691.

Reply:
ok I hope we can fix that later then!